src/System.Linq.Expressions/tests/CompilerTests.cs

```diff
@@ -27,7 +27,7 @@ public static void CompileDeepTree_NoStackOverflow(bool useInterpreter)
             Assert.Equal(n, f());
         }
 
-        [Theory, ActiveIssue("https://github.com/dotnet/corefx/issues/15533"), ClassData(typeof(CompilationTypes))]
+        [Theory, ClassData(typeof(CompilationTypes))]
```

Comment:
Actually, I don't think this test is valid any more.  The runtime no longer puts a lower bound on stack size, so if you create a stack of size 1, it's basically guaranteed that it'll stack overflow as part of just trying to execute the code in the test that would check stack depth.